public/src/Helpers.ts

```diff
@@ -263,20 +263,19 @@ export const defaultYear = (date: Date): number => {
     return (date.getMonth() <= 11 && date.getDate() <= 31) && date.getMonth() > 10 ? (date.getFullYear() + 1) : date.getFullYear()
 }
 
-export const userLink = (user: User.AsObject): string => {
+export const userLink = (user: User): string => {
     return `https://github.com/${user.login}`
 }
 
-export const userRepoLink = (course: Course.AsObject, user: User.AsObject): string => {
-    return `https://github.com/${course.organizationname}/${user.login}-labs`
+export const userRepoLink = (course: Course, user: User): string => {
+    return `https://github.com/${course.organizationName}/${user.login}-labs`
 }
 
-export const groupRepoLink = (course: Course.AsObject, group: Group.AsObject): string => {
-    course.organizationname
-    return `https://github.com/${course.organizationname}/${slugify(group.name)}`
+export const groupRepoLink = (course: Course, group: Group): string => {
+    return `https://github.com/${course.organizationName}/${slugify(group.name)}`
```

Comment:
Do we need the `slugify` here? I don't think we allow a `Group` object to be saved in the database with an illegal group name.